index.bs

```diff
@@ -318,6 +339,285 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
+      <td>2</td>
+      <td>negate operator</td>
+      <td>1</td>
+      <td>Negation of the left operand.</td>
+      <td><math><mo>-</mo><mi>L</mi></math></td>
+    </tr>
+    <tr>
+      <td>3</td>
+      <td>absolute value operator</td>
+      <td>1</td>
+      <td>Absolute value of the left operand.</td>
+      <td><math><mo>|</mo><mi>L</mi><mo>|</mo></math></td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>not operator</td>
+      <td>1</td>
+      <td>Bitwise complement of the operand.</td>
+      <td><math><mo>¬</mo><mi>L</mi></math></td>
+    </tr>
+    <tr>
+      <td>5</td>
+      <td>bsr operator</td>
+      <td>1</td>
+      <td>0-based index of the most significant set bit of the left operand if the left operand is strictly positive, zero otherwise.</td>
+      <td>
+        <math>
+          <mo>{</mo>
+          <mtable>
+            <mtr><mtd><mn>0</mn></mtd><mtd><mo>if</mo><mi>L</mi><mo>≤</mo><mn>0</mn></mtd></mtr>
+            <mtr><mtd>truncate<mo>(</mo><msub><mo>log</mo><mn>2</mn></msub><mi>L</mi><mo>)</mo></mtd><mtd>otherwise</mtd></mtr>
+          </mtable>
+        </math>
+      </td>
+    </tr>
+    <tr>
+      <td>128</td>
+      <td>sum operator</td>
+      <td>2</td>
+      <td>Left operand added to the right operand.</td>
+      <td><math><mi>L</mi><mo>+</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>129</td>
+      <td>difference operator</td>
+      <td>2</td>
+      <td>Right operand subtracted from the left operand.</td>
+      <td><math><mi>L</mi><mo>-</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>130</td>
+      <td>product operator</td>
+      <td>2</td>
+      <td>Left operand multiplied by the right operand.</td>
+      <td><math><mi>L</mi><mo>×</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>131</td>
+      <td>divide operator</td>
+      <td>2</td>
+      <td>Left operand divided by the right operand if the right operand is not zero, left operand otherwise. The result is truncated toward zero (integer division).</td>
+      <td>
+        <math>
+          <mo>{</mo>
+          <mtable>
+            <mtr><mtd><mi>L</mi></mtd><mtd><mo>if</mo><mi>R</mi><mo>=</mo><mn>0</mn></mtd></mtr>
+            <mtr><mtd>truncate<mo>(</mo><mfrac><mi>L</mi><mi>R</mi></mfrac><mo>)</mo></mtd><mtd>otherwise</mtd></mtr>
+          </mtable>
+        </math>
+      </td>
+    </tr>
+    <tr>
+      <td>132</td>
+      <td>and operator</td>
+      <td>2</td>
+      <td>Bitwise conjunction of the operands.</td>
+      <td><math><mi>L</mi><mo>∧</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>133</td>
+      <td>or operator</td>
+      <td>2</td>
+      <td>Bitwise inclusive disjunction of the operands.</td>
+      <td><math><mi>L</mi><mo>∨</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>134</td>
+      <td>xor operator</td>
+      <td>2</td>
+      <td>Bitwise exclusive disjunction of the operands.</td>
+      <td><math><mi>L</mi><mo>⊻</mo><mi>R</mi></math></td>
```

Comment:
This may be a regional difference. I have seen the symbol ⊕ used for the xor operator.